src/actions/video.js

```diff
@@ -24,6 +26,61 @@ export const LOADED_DATA = 'video-react/LOADED_DATA';
 export const RESIZE = 'video-react/RESIZE';
 export const ERROR = 'video-react/ERROR';
 
+export const Actions = generateActionsEnum([
+  LOAD_START,
+  CAN_PLAY,
+  WAITING,
+  CAN_PLAY_THROUGH,
+  PLAYING,
+  PLAY,
+  PAUSE,
+  END,
+  SEEKING,
+  SEEKED,
+  SEEKING_TIME,
+  END_SEEKING,
+  DURATION_CHANGE,
+  TIME_UPDATE,
+  VOLUME_CHANGE,
+  PROGRESS_CHANGE,
+  RATE_CHANGE,
+  SUSPEND,
+  ABORT,
+  EMPTIED,
+  STALLED,
+  LOADED_META_DATA,
+  LOADED_DATA,
+  RESIZE,
+  ERROR
+]);
+
+export const ActionCreators = {
```

Comment:
I don't think we really need this,
We could use 
```
import  * as playerActions from './actions/player';
```
instead of
```
import { ActionCreators as PlayerActionCreators } from './actions/player';
```